host/hackrf-tools/src/hackrf_transfer.c

```diff
@@ -1058,20 +1081,33 @@ int main(int argc, char** argv) {
 		    }
 #endif
 		} else {
+		        uint64_t	stream_amplitude_now;
 			sleep(1);
 			gettimeofday(&time_now, NULL);
 			
 			byte_count_now = byte_count;
 			byte_count = 0;
-			
-			
+		        stream_amplitude_now = stream_amplitude;
+		        stream_amplitude = 0;
+			if (byte_count_now < sample_rate_hz/20)	// Don't report on very short frames
+				stream_amplitude_now = 0;
+
 			time_difference = TimevalDiff(&time_now, &time_start);
 			rate = (float)byte_count_now / time_difference;
 			if (byte_count_now == 0 && hw_sync == true && hw_sync_enable != 0) {
 			    fprintf(stderr, "Waiting for sync...\n");
 			} else {
-			    fprintf(stderr, "%4.1f MiB / %5.3f sec = %4.1f MiB/second\n",
-					    (byte_count_now / 1e6f), time_difference, (rate / 1e6f) );
+			    // This is only an approximate measure, to assist getting receive levels right:
+			    double	full_scale_ratio = ((double)stream_amplitude_now / (byte_count_now ? byte_count_now : 1))/128;
+			    double	dB_full_scale_ratio = 10*log(full_scale_ratio)/M_LN10;
```

Comment:
Is there a reason not to use `log10` here? That would also remove the need for the special-case defining the LN10 constant.

Also, do we need to use double precision for this?